packages/datepicker/src/types/prop-getters.ts

```diff
@@ -2,6 +2,7 @@ import { MouseEvent } from 'react';
 
 export interface DPPropsGetterConfig extends Record<string, unknown> {
   onClick?(evt?: MouseEvent<HTMLElement>, date?: Date): void;
+  onPress?(evt?: any, date?: Date): void;
```

Comment:
`any` is not acceptable.
Why we need to add additional prop on top of `onClick` for ReactNative. Why can't we pass `onClick: onPress`?

Reply:
just wanted to avoid dependency to react native.
or maybe  we can copy `GestureResponderEvent` from react native ?

Reply:
onClick is not working in react native so using onPress is common, although here onClick will also work. but to avoid confusion included onPress as well.

Reply:
Better to add react-native to the pearDependencies and specify type.